engine/src/main_loop.cpp

```diff
@@ -792,22 +791,27 @@ void createObjects(std::vector<std::string> &fighter0name,
     vector<std::string> fighter0mods;
     vector<int> fighter0indices;
 
-    static Vector TerrainScale(game_options()->xscale, game_options()->yscale, game_options()->zscale);
+    static Vector * terrain_scale;
```

Comment:
please ensure this is set to `nullptr`; even though we're using `initialized` too.

Reply:
Done